### ISSUE TITLE
mugration / treetime sequence storage error

### ISSUE THREAD

**Issue by jameshadfield** [nextstrain/augur#21]
There is a treetime bug introduced between
`4020af46fa00feaa0a081d550eaccb2f0df9f4f0` (currently used for `augur/master`) and treetime `383ec6ffd0a777047e14d6289c8353081c8c5655` (currently used for `augur/rejig`).

Treetime & vanilla mugration model works fine but specifying a `root_state` causes the following traceback:
```
  File "zika/zika.py", line 132, in <module>
    zika.tree.geo_inference('region', root_state = 'southeast_asia')
  File "./base/tree.py", line 284, in geo_inference
    store_compressed=False, pc=5.0, marginal=True, normalized_rate=False)
  File "../treetime_augur/treetime/treeanc.py", line 320, in infer_ancestral_sequences
  File "../treetime_augur/treetime/treeanc.py", line 347, in reconstruct_anc
  File "../treetime_augur/treetime/treeanc.py", line 559, in _ml_anc_marginal
AttributeError: 'Clade' object has no attribute 'cseq'
```

I'm guessing this is due to the compressed sequences storage being incompatable with how the "extra_clade" is defined https://github.com/nextstrain/augur/blob/master/base/tree.py#L277

how to reproduce: update treetime in augur/master & build zika

cc @rneher

**Comment by rneher** [comment]
this was fixed in 91ea4cd093bc75f9d875da5b58a8537e66804cef but must have been lost in subsequent merges

**Comment by rneher** [comment]
I could add a `reassign_sequences()` to treetime to avoid such problems -- the geo_inference procedure is messing around with many internals.

**Comment by jameshadfield** [comment]
Fixed in 20bc0ebf5dd1a71ff10dfffb498e5355107c3fca

👍  @rneher
